assets/youtube_safe_search.txt

```diff
@@ -5,3 +5,8 @@
 |www.youtube.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
 |youtube.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
 |youtubei.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|www.youtube.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|m.youtube.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|youtubei.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|youtube.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|www.youtube-nocookie.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
```

Comment:
Literally the same rules

Reply:
Lol)
 Maybe @Mizzick want us to duplicate them in engines_safe_search.txt?

Reply:
Indeed I wanted these rules in `engines_safe_search.txt` as I haven't seen that you have them already in the `youtube_safe_search.txt` file.

Then it's fine, no need to change anything.